common/src/main/java/com/wynntils/wynn/utils/WynnBossBarUtils.java

```diff
@@ -111,5 +113,21 @@ public static BarProgress getFocusBar() {
         }
     }
 
+    public static BarProgress getCorruptedBar() {
+        Pair<LerpingBossEvent, Matcher> pair = getLerpingBossEvent(CORRUPTED_PROGRESS_PATTERN);
+
+        if (pair.a == null) return NO_CORRUPTED;
+
+        try {
+            int currentCorruptedProgress = Integer.parseInt(pair.b.group(1));
+            int maxCorruptedProgress = 100;
```

Comment:
Can we make this a const in the class?

I am sad we cannot parse this... Or can we? From what I see in the regex, isn't this like blood pool, where we can use the lerpingBossEvent to parse this "better"? Although, if it is a 0-100% range, we might not need that..

Reply:
It is 0-100% range